src/components/List/list.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Styles from "./list.module.css";
+
+type TodoItem = {
+  id: string;
+  title: string;
+};
+
+type ListProps = {
+  data: TodoItem[];
+};
+
+const List: React.FC<ListProps> = ({ data }) => {
```

Comment:
With the latest React and Typescript updates, React.FC is not needed anymore and it's actually discouraged. A component should be defined like this now:

```
interface MyComponentProps {
  // props
}

function MyComponent({name}: MyComponentProps) {
  // render logic 
}
```